x/dex/keeper/core.go

```diff
@@ -25,6 +25,7 @@ func (k Keeper) DepositCore(
 	callerAddr sdk.AccAddress,
 	amounts0 []sdk.Int,
 	amounts1 []sdk.Int,
+	autoswap bool,
```

Comment:
autoswap is already on msg, no need to pass this in

Reply:
Ahhhh I see now. When I saw token0,1 and the other fields there I replicated it with autoswap but realize now that this was just bc they need to be sorted or the type needs to change.